include/oneapi/dpl/pstl/ranges/nanorange.hpp

```diff
@@ -103,6 +103,8 @@
 #                                define NANO_CONCEPT inline constexpr bool
 
 #                                define NANO_BEGIN_NAMESPACE                                                           \
+                                    namespace __nanorange                                                              \
+                                    {                                                                                  \
                                                                                                                        \
                                     namespace nano                                                                     \
```

Comment:
Are there benefits of adding an extra namespace "above" `namespace nano`, instead of simply renaming it to e.g. dpl_nano?

Reply:
The minimal (as possible) changes with the current code and the code of NanoRange library in case of sync.